src/index.html

```diff
@@ -5,8 +5,389 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header
+    class="top-bar"
+    id="home"
+    >
+      <a href="#home">
+        <img
+        class="logo"
+        src="./images/logo.png"
+        alt="logothype"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
```

Comment:
add an empty line only between **_multiline_** siblings. 
And no needs to add this line between _**oneline**_ siblings
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>
```